docs/storage/storage.md

```diff
@@ -77,11 +126,11 @@ export class AppComponent {
 
 An `AngularFireUploadTask` has methods for observing upload percentage as well as the final download URL.
 
-| method   |                    | 
-| ---------|--------------------| 
-| `snapshotChanges(): Observable<FirebaseStorage.UploadTaskSnapshot>` | Emits the raw `UploadTaskSnapshot` as the file upload progresses. | 
-| `percentageChanges(): Observable<number>` | Emits the upload completion percentage. | 
-| `downloadURL(): Observable<string>` | Emits the download url when available |
+| method   |                    |
+| ---------|--------------------|
+| `snapshotChanges(): Observable<FirebaseStorage.UploadTaskSnapshot>` | Emits the raw `UploadTaskSnapshot` as the file upload progresses. |
+| `percentageChanges(): Observable<number>` | Emits the upload completion percentage. |
+| `getDownloadURL(): Observable<string>` | Emits the download url when available |
```

Comment:
I'd updated the example with a more thorough description of this change, moreover, I've noticed that the type of the Observable changed to `getDownloadURL(): Observable<any>`,  instead of  `getDownloadURL(): Observable<string>`, thus I've matched the new declaration at the docs. [storage ref - line 7](https://github.com/angular/angularfire2/blob/master/src/storage/ref.ts)